lib/resource-locker.libjsonnet

```diff
@@ -98,28 +100,35 @@ local replaceColon(str) =
 local rl_obj_name(objdata) =
   // Some objects like ClusterRoleBinding can contain colons.
   local name = replaceColon(objdata.name);
```

Comment:
Not actually part of the PR, but I just realized this might fail for some OpenShift resources, which allow all kinds of names.

Reply:
I noticed this as well, but I don't think it's something we have to worry about at the moment, as most objects we patch have relatively "normal" names.